src/ai/susi/server/api/aaa/ModifyUserDevices.java

```diff
@@ -98,7 +98,7 @@ public ServiceResponse serviceImpl(Query call, HttpServletResponse response, Aut
                             }
                         }
                         else {
-                            throw new APIException(400, "Specified device does not exist.");
```

Comment:
I think it should also not be a 404

As the device name which was provided as query string is not present

So the error code should be `422` or `400`